cli/logout_test.go

```diff
@@ -152,19 +152,19 @@ func TestLogout(t *testing.T) {
 			err = os.Chmod(string(config), 0500)
 			require.NoError(t, err)
 		}
-		t.Cleanup(func() {
+		defer func() {
 			if runtime.GOOS == "windows" {
 				// Closing the opened files for cleanup.
 				err = urlFile.Close()
-				require.NoError(t, err)
+				assert.NoError(t, err)
```

Comment:
Assert here, to avoid interrupting `Close` below.

Reply:
Should comment this in the file so it doesn't get reverted

Reply:
I kind of agree, but I'm less worried about this old code changing than new (incorrect) code going in. This is something everyone using `require` must be very aware of (it halts execution flow).